pilot/pkg/serviceregistry/kube/controller/ambientindex_test.go

```diff
@@ -372,18 +372,22 @@ func TestAmbientIndex_Policy(t *testing.T) {
 	s := newAmbientTestServer(t, testC, testNW)
 
 	s.addPods(t, "127.0.0.1", "pod1", "sa1", map[string]string{"app": "a"}, nil, true, corev1.PodRunning)
+	s.assertEvent(t, s.podXdsName("pod1"))
```

Comment:
not sure how https://github.com/istio/istio/pull/45797 merged, but after it merged ci and my local tests were failing until i added the missing events

cc @nmittler @howardjohn

Reply:
this has been handled by https://github.com/istio/istio/pull/45925 and thus removed after merging this with latest master